docs/en/oas3/OAS3-PDND-WP.yaml

```
@@ -0,0 +1,299 @@
+openapi: 3.0.1
+info:
+  title: IT Wallet API - Wallet Provider web services
+  version: 0.1.0
+servers:
+  - url: https://wallet-provider.example.org
+    description: Wallet Provider API
+paths:
+  /v1.0.0/wallet-instances/:
```

Better to manage the version at the base url level
```suggestion
servers:
  - url: https://pdnd.wallet-provider.example.org/api/v1.0
    description: Wallet Provider API for PDND
paths:
  /wallet-instances:
```